### PR TITLE
add NTP configuration in esx-ks

### PR DESCRIPTION
**Pull request by lanchongyizu**
**Fix ODR-750** NTP configuration in esx-ks

The solution is adding ntpSevers options to support NTP configuration, so that the users could use the private NTP servers if needed.

@RackHD/corecommitters @panpan0000

### CODE REVIEW

File: lib/task-data/tasks/install-esx.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ module.exports = {
         users: [],
         networkDevices: [],
         dnsServers: [],
+        ntpServers: [],
```
Comment (yyscamper) [inline lib/task-data/tasks/install-esx.js]: After the task-schema upfront validation is enabled, this line should not put here. see this PR: https://github.com/RackHD/on-tasks/pull/257

**Comment by yyscamper** [comment]
There will be some conflict after the task-schema upfront validation is enabled (see https://github.com/RackHD/on-tasks/pull/257), I suggest you waiting that PR is merged then resolve the conflict, because that PR is very complex, but yours is relative simple ^^

**Comment by lanchongyizu** [comment]
@yyscamper OK, please remind me after the #257 is merged, thanks.

**Comment by anhou** [comment]
@lanchongyizu also don't forget a PR for docs

**Comment by lanchongyizu** [comment]
test this please

**Comment by brianparry** [comment]
test this please

**Pull request closed without merging**